doc/Getting-started/index.md

```diff
@@ -65,543 +41,211 @@ To use Kaa Sandbox, your system must meet the following minimum system requireme
-
-1. Open the relevant **Notification topics** window (**Applications =\> My First Kaa Application =\> Notification topics**) and click **Add new notification topic**.
-2. In the **Add notification topic** window, fill in the fields as shown in the following screenshot and then click **Add**.  
-![](attach/image2014-7-28 13-38-3.png)
-
-**NOTE:** We set the topic as mandatory in order to automatically subscribe the client application to notifications on this topic.
-
-Once the topic is created, we will assign it to the default endpoint group, which contains all endpoints, including endpoints with our application.
-
-To assign a notification topic to the default endpoint group, proceed as follows:
-
-1. In the relevant **Endpoint groups** window (**Applications=\>My First Kaa Application=\>Endpoint groups**), select the _All_ group.
-2. In the **Endpoint group details** window, click **Add notification topic** at the bottom of the window.
-3. In the **Add topic to endpoint group** window, select the recently created notification topic and then click **Add**.  
-![](attach/image2014-7-28 13-43-38.png)
-
-After the topic is added to the endpoint group, you will see the following output in the application:
-
-    [...  INFO  NotificationSystemTestApp]  Topic list updated!
-    [...  INFO  NotificationSystemTestApp]  Received topic with id X and name Notification Topic
+</div></div>
 
-This is the first update from the Kaa server that provides the client with the information about the changes in the topic list. Now you can send notifications on this topic, and they will be delivered to your application.
+# Launching your first Kaa application
 
-## Create notification
+So let's run your first Kaa aplication. Navigate to Demo projects page and select **Data collection demos** application bundle.
 
-To create a notification, proceed as follows:
+<br>
 
-1. Create a _notification.json_ file on your PC with the following contents:
+<p align="center">
+  <img src="attach/sandbox_first_app_bundle_view.png" width="500" height="400">
+</p>
 
-    {"message":"Hello from Kaa!"}
+<br>
 
-2. Open the relevant **Notification topics** window (**Applications =\> My First Kaa Application =\> Notification topics**) and click **Send notification** in the **Notification Topic** row.  
-<img src="attach/image2015-6-19 14-51-26.png" width="800" height="500">
-3. In the **Send notification** window, fill in the fields as shown in the following screenshot and then click **Send**.  
-![](attach/image2014-7-28 13-52-48.png)
+Than select application for your platform, download sources or binaries and follow instructions provided in application description.
 
-Once the notification is sent, you can see the following output in the application:
+<br>
 
-    [...  INFO  NotificationSystemTestApp]  Received notification {"message": "Hello from Kaa!"} for topic with id X
+<p align="center">
+  <img src="attach/sandbox_first_app_view.png" width="500" height="400">
```

Comment:
Not a cool way to insert images:
- centering is hard-coded
- size is hard-coded

Both things should be done with css styles.